lib/tasks/broadcasts.rake

```diff
@@ -0,0 +1,6 @@
+desc "Send Welcome Notifications"
+task broadcast_welcome_notification_flow: :environment do
+  User.where("created_at < ?", 7.days.ago).find_each do |user|
```

Comment:
Should we use a specific timestamp here instead of `7.days.ago`? Just wondering if we want a more granular sense of when this feature went "live" and when we started sending out these notifications 🤔

Reply:
Also...if someone's account was created 6 days ago, will they receive all the notifications at once? It seems like a user created 6 days ago would suddenly get all the notifications, once every hour, which is probably not the behavior we want

Reply:
I implemented a solution to address that concern